packages/cli/config/schema.ts

```diff
@@ -577,6 +577,12 @@ export const schema = {
 	},
 
 	publicApiEndpoints: {
+		disable: {
+			format: Boolean,
+			default: false,
+			env: 'N8N_PUBLIC_API_DISABLE',
```

Comment:
following the naming convention (see `N8N_DISABLE_UI` above), this env var should be called `N8N_DISABLE_PUBLIC_API`